### ISSUE TITLE
Additional charts

### ISSUE THREAD

**Issue by nijel** [phpmyadmin/error-reporting-server#65]
The stats page could include more charts to show evolution of incidents in time.
- phpMyAdmin versions over time
- PHP versions over time
- Groupped browsers (ignoring version)

**Comment by Dhrumil1808** [comment]
@nijel  Do you want the statistics to be displayed differently for different versions of PhpMyAdmin and PHP? Changes need to be done in status tab which includes Processes, query statistics, status variables etc? Am I Right?

**Comment by nijel** [comment]
That could be useful as well, but my original idea was simpler - provide line chart with phpMyAdmin versions, to be able to see adoption over time. Same for PHP versions. For browsers it would be pie chart just showing browser name without version.

**Comment by Dhrumil1808** [comment]
So, according to you, in the line chart different PHP versions should be displayed on the X -axis. Then which parameters should be considered on the Y-axis? 
Should the changes be done in the status tab in the monitor section?

**Comment by nijel** [comment]
No, the X axis would be time, Y axis number of incidents and versions would be individual chart series.

Something like [this](https://en.wikipedia.org/wiki/Comparison_of_web_browsers#/media/File:Usage_share_of_web_browsers_%28Source_StatCounter%29.svg), just for PHP and phpMyAdmin versions. Hmm actually the browser chart would also make sense...

**Comment by Dhrumil1808** [comment]
In the monitor section, Connections/processes would contain a line chart showing the performance of different versions of phpmyadmin and PHP over time. Similarly this holds true for traffic, System CPU usage ,System Memory and System Wrap too. Please correct me if my understanding is wrong.
Basically, the changes need to be done in the monitor section of status tab?

**Comment by nijel** [comment]
No this is not about phpMyAdmin itself, it's about error reporting server we use for collecting error reports. It's completely separate code base living in this repository...

**Comment by nijel** [comment]
To make it clear, here is screenshot how stats look currently:

![stats_phpmyadmin_-_error_reporting_server_-_2016-01-18_12 04 21](https://cloud.githubusercontent.com/assets/212189/12389827/a0fb0ee8-bddb-11e5-94fb-457fbc03a497.png)

**Comment by Dhrumil1808** [comment]
Can I take this project for GSOC 2016?

**Comment by lem9** [comment]
@Dhrumil1808 Have a look at http://wiki.phpmyadmin.net/pma/GSoC_2016_Ideas_List#Error_reporting_server.

**Comment by Dhrumil1808** [comment]
In the error-reporting server, none of the links "Reports", "Stats" etc. are working. Also,I am unable to login into the GitHub Account through error-reporting-server. How to connect this with phpmyadmin?

**Comment by nijel** [comment]
You need to run server locally to have access to these. The production one is accessible only for team members....

**Comment by MaleicAcid** [comment]
Ah ~ It seems there are a lot of charts now, what additional statistics need to be added?

**Comment by nijel** [comment]
I think the initial comment  https://github.com/phpmyadmin/error-reporting-server/issues/65#issue-51076286 still applies, none of the charts are there and they would be useful to see trending (eg. that some version is more buggy than other).
